test/features/parsoid/transform.js

```diff
@@ -15,197 +16,166 @@ describe('transform api', function() {
             assert.deepEqual(res.body, []);
         });
     });
 
     it('wt2lint with errors', () => {
-        return preq.post({
-            uri: `${server.config.baseURL('en.wikipedia.beta.wmflabs.org')}/transform/wikitext/to/lint`,
-            body: {
-                wikitext: '<div>No div ending'
-            }
-        }).then((res) => {
+        return client.post(
+            `${server.config.baseURL('en.wikipedia.beta.wmflabs.org')}/transform/wikitext/to/lint`,
+            { wikitext: '<div>No div ending' }
+        ).then((res) => {
             assert.deepEqual(res.status, 200);
             assert.deepEqual(res.body.length, 1);
         });
     });
 
     it('html2wt, no-selser', () => {
-        return preq.post({
-            uri: `${server.config.baseURL('en.wikipedia.beta.wmflabs.org')}/transform/html/to/wikitext/${testPage.title}`,
-            body: {
-                html: '<body>The modified HTML</body>'
-            }
+        return client.post(
+            `${server.config.baseURL('en.wikipedia.beta.wmflabs.org')}/transform/html/to/wikitext/${testPage.title}`,
+            { html: '<body>The modified HTML</body>' }
 
-        })
+        )
         .then((res) => {
             assert.deepEqual(res.status, 200);
-            assert.deepEqual(res.body, 'The modified HTML');
-            assert.contentType(res, contentTypes.wikitext);
+            assert.deepEqual(res.text, 'The modified HTML');
+            assert.match(res.headers['content-type'], mwUtil.constructRegex([contentTypes.wikitext]));
         });
     });
 
     it('html2wt, selser', () => {
-        return preq.post({
-            uri: `${server.config.baseURL('en.wikipedia.beta.wmflabs.org')}/transform/html/to/wikitext/${testPage.title}/${testPage.revision}`,
-            body: {
-                html: testPage.html
-            }
-        })
+        return client.post(
+            `${server.config.baseURL('en.wikipedia.beta.wmflabs.org')}/transform/html/to/wikitext/${testPage.title}/${testPage.revision}`,
+            { html: testPage.html }
+        )
         .then((res) => {
             assert.deepEqual(res.status, 200);
-            assert.deepEqual(res.body, testPage.wikitext);
-            assert.contentType(res, contentTypes.wikitext);
+            assert.deepEqual(res.text, testPage.wikitext);
+            assert.match(res.headers['content-type'], mwUtil.constructRegex([contentTypes.wikitext]));
         });
     });
 
     it('html2wt with scrub_wikitext', () => {
-        return preq.post({
-            uri: `${server.config.baseURL('en.wikipedia.beta.wmflabs.org')}/transform/html/to/wikitext`,
-            body: {
-                html: '<h2></h2>',
-                scrub_wikitext: 1
-            }
-        })
+        return client.post(
+            `${server.config.baseURL('en.wikipedia.beta.wmflabs.org')}/transform/html/to/wikitext`,
+            { html: '<h2></h2>', scrub_wikitext: 1 }
```

Comment:
Change 1 to 'true' - this parameter should be a boolean, and preq was automatically doing a conversion while api-testing doesn't.

Reply:
Thanks @Pchelolo, I have fixed the `html2wt with scrub_wikitext` test as per your advice.